lib/cloudflare.ts

```diff
@@ -1,23 +1,23 @@
 // See https://developers.cloudflare.com/turnstile/
-const secret: string = process.env.CLOUDFLARE_SECRET as string;
+// const secret: string = process.env.CLOUDFLARE_SECRET as string;
 
-const verifyEndpoint =
-  "https://challenges.cloudflare.com/turnstile/v0/siteverify";
+// const verifyEndpoint =
+//   "https://challenges.cloudflare.com/turnstile/v0/siteverify";
 
-export const checkCloudflare = async (
-  cloudflareCallback: string
-): Promise<boolean> => {
-  const cloudflareResponse = await fetch(verifyEndpoint, {
-    method: "POST",
-    body: `secret=${encodeURIComponent(secret)}&response=${encodeURIComponent(
-      cloudflareCallback
-    )}`,
-    headers: {
-      "content-type": "application/x-www-form-urlencoded",
-    },
-  });
+// export const checkCloudflare = async (
```

Comment:
Are we planning to add Cloudflare back at some point? How come this section is commented out while the other Cloudflare sections are deleted?